ArtisansFinal/.idea/dictionaries/shrinidhi.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="shrinidhi">
```

Comment:
Why is this file here?

Reply:
i have finally found the file. Two ways of removing it. One is go to the gitignore file of the project and add a line of code there which looks the best option. Another option is to delete the file from the project folder.
which one do you recommend?

Reply:
/.idea/dictionaries.xml
this is the line to be added